src/textual/css/_styles_builder.py

```diff
@@ -770,6 +771,18 @@ def process_align_vertical(self, name: str, tokens: list[Token]) -> None:
     process_content_align_horizontal = process_align_horizontal
     process_content_align_vertical = process_align_vertical
 
+    def process_scrollbar_gutter(self, name: str, tokens: list[Token]) -> None:
```

Comment:
it was interesting to see all the different steps of adding a new CSS property in Textual :slightly_smiling_face: